cmake/extensions.cmake

```diff
@@ -20,19 +20,34 @@ function(get_board_without_ns_suffix board_in board_out)
   endif()
 endfunction()
 
-# Add a kconfig overlay file to a child image.
-# This can be used by a parent image to set kconfig values in its child images.
-# This must be invoked before 'add_child_image(image)'
-function(add_overlay_config image overlay_file)
-  set(old_conf ${${image}_OVERLAY_CONFIG})
-  string(FIND "${old_conf}" "${overlay_file}" found)
+# Add an overlay file to a child image.
+# This can be used by a parent image to set configuration or device tree overlay
```

Comment:
The terminology is not easy to get right here, I guess @tejlmand  might have some wishes.

Reply:
I've slightly modified the description here. I hope it is a bit more straightforward now.